rust/release/src/main.rs

```diff
@@ -352,23 +352,32 @@ pub const DATABANK_VERSION: &str = "{}";
 
                             // Update the ax_core version in ax
                             eprintln!("0.3.2) Updating the ax_core version in \"{}\" ... ", ax_cargo.display());
-                            update_dependent_version(&ax_cargo, &new_version, "ax_core").context(format!(
-                                "updating {} to {}",
-                                &ax_cargo.display(),
-                                &new_version
-                            ))?;
+                            {
+                                let cargo_toml_contents = std::fs::read_to_string(&ax_cargo)?;
+                                let mut cargo_toml = cargo_toml_contents.parse::<Document>()?;
+                                // Similar to `update_package_version` but in this case,
+                                // we're updating the version of a crate that should depend "on us"
+                                cargo_toml["dependencies"]["ax_core"]["version"] =
+                                    toml_edit::value(&new_version.to_string());
+
+                                cargo_toml["build-dependencies"]["ax_core"]["version"] =
+                                    toml_edit::value(&new_version.to_string());
+
+                                std::fs::write(&ax_cargo, cargo_toml.to_string())?;
+                            }
                             repo.add_file(&ax_cargo)
                                 .context(format!("adding {} to repo", &ax_cargo.display()))?;
 
                             // Update the lockfile
                             eprintln!("0.3.3) Updating the lockfile");
                             std::process::Command::new("cargo")
+                                .current_dir(rust_folder.join("actyx/ax"))
                                 .arg("update")
                                 .output()
                                 .expect("failed to execute `cargo update`");
-                            let lockfile = rust_folder.join("Cargo.lock");
-                            repo.add_file(lockfile)
-                                .context(format!("adding {} to repo", &ax_cargo.display()))?;
+                            let lockfile = rust_folder.join(PathBuf::from("actyx/Cargo.lock"));
```

Comment:
why the PathBuf? according to the rust docs `.join("some string")` should work

Reply:
It does, it was a lapse